### PR TITLE
Implement ES proxy in charmstore

### PR DESCRIPTION
**Pull request by jrwren**
Implement elasticsearch proxy endpoint in charmstore. Will be useful
in an environment where elasticsearch is unreachable.

https://canonical.leankit.com/Boards/View/102529849/111347878

### CODE REVIEW

**Comment by jujugui** [comment]
Test FAILed.
Refer to this link for build results: http://ci.jujugui.org:8080/job/charmstore/336/

**Comment by jujugui** [comment]
Test PASSed.
Refer to this link for build results: http://ci.jujugui.org:8080/job/charmstore/337/

**Comment by jujugui** [comment]
Test PASSed.
Refer to this link for build results: http://ci.jujugui.org:8080/job/charmstore/338/

**Comment by jujugui** [comment]
Test PASSed.
Refer to this link for build results: http://ci.jujugui.org:8080/job/charmstore/339/

File: internal/elasticsearch/test/elasticsearch.go [hunk 1]
```diff
@@ -0,0 +1,4 @@
+package elasticsearch
```
Comment (mhilton) [inline internal/elasticsearch/test/elasticsearch.go:1]: You've fixed  this the other  way now.

**Comment by mhilton** [comment]
:+1: Looks fine

**Pull request closed without merging**